### PR TITLE
Home task14

### PR DESCRIPTION
**Pull request by aliaksandr-harunou**

### CODE REVIEW

File: test/features/cart.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    When I choose "Большая" pizza size
+    When I move current pizza to the cart
+    When I open cart
+    When I wait 1 seconds
+    Then Count of selected pizza should be equal to 2
+    And I wait 1 seconds
+    When I delete first pizza from the cart
+    And I wait 1 seconds
+    Then Count of selected pizza should be equal to 1
+    And I wait 1 seconds
```
Comment (VladSelikh) [inline test/features/cart.feature]: Please remove this line from the scenario.

File: test/features/navigation.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@smoke
+@navigation
+Feature: Navigation
+
+
+Scenario Outline: Should scroll to "<navigationLink>"
+    Given I open pzz.by
+    When I navigate to "<navigationLink>" using navigation menu
+    And I wait 1 seconds
```
Comment (VladSelikh) [inline test/features/navigation.feature]: Please remove this line from the scenario.

File: test/step_definitions/hooks.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const { After } = require('@cucumber/cucumber');
+const { browser } = require('protractor');
+
+
+After(async function () {
+    const screenshot = await browser.takeScreenshot();
+    const decodedImage = new Buffer.from(screenshot, 'base64');
+    return this.attach(decodedImage, 'image/png'); 
+});
```
Comment (VladSelikh) [inline test/step_definitions/hooks.js:11]: Please add a hook to attach screenshot if the step fails.

**Comment by VladSelikh** [comment]
Please don't commit files within reports folder.

File: test/features/cart.feature (hunk 1 above)
Comment (aliaksandr-harunou) [reply test/features/cart.feature]: removed

File: test/features/navigation.feature (hunk 2 above)
Comment (aliaksandr-harunou) [reply test/features/navigation.feature]: removed

File: test/step_definitions/hooks.js (hunk 3 above)
Comment (aliaksandr-harunou) [reply test/step_definitions/hooks.js:11]: improved

**Pull request merged**